src/app/app.component.html

```diff
@@ -22,25 +22,25 @@
 
           <md-content layout-padding class="submenuarea">
             <md-list class="sidesubmenu">
-              <a routerLinkActive="active" routerLink="/wallet/receive">
+              <a routerLinkActive="active" [routerLink]="(!isWalletInitialized) ? null : '/wallet/receive'">
```

Comment:
This is really untidy.. Rather disable the links, that way the users get a cursor informing them the link is disabled.

You can use a css class that has
```css
    pointer-events: none;
    cursor: not-allowed;
```